docs/internal/docker.md

```
@@ -0,0 +1,273 @@
+
+```bash
+ls logs/e2e-*.log              # Entrypoint pass/fail output
+ls logs/workflow-*.jsonl       # claude -p JSON output
+ls logs/debug-*.log            # Claude Code debug logs
+```
+
+### Debugging Plugin Issues
+
+1. **Hook errors** (`SessionStart:startup hook error`): Check the debug log for the hook that failed. Common causes: missing files in the plugin directory, permission issues, or incompatible plugin format
+2. **Plugin not loading**: Verify the plugin directory structure inside the container:
+   ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)